### PR TITLE
Use simplified download URL for sourceforge.

### PR DESCRIPTION
**Pull request by MikaelSmith**

### CODE REVIEW

File: manual/mingw/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -50,10 +50,10 @@ if ($exception -eq 'sjlj') {
...
-#http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/4.8.3/threads-posix/sjlj/i686-4.8.3-release-posix-sjlj-rt_v3-rev0.7z
-$url64 = "http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/$packageVersion/threads-$threads/$exceptionHandling64/x86_64-$packageVersion-release-$threads-$exceptionHandling64-rt_v3-$rev.7z"
-#http://downloads.sourceforge.net/project/mingw-w64/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/4.8.3/threads-posix/sjlj/x86_64-4.8.3-release-posix-sjlj-rev0.7z
+$url = "http://downloads.sourceforge.net/mingw-w64/i686-$packageVersion-release-$threads-$exceptionHandling-rt_v3-$rev.7z"
+#http://downloads.sourceforge.net/mingw-w64/i686-4.8.3-release-posix-sjlj-rt_v3-rev0.7z
+$url64 = "http://downloads.sourceforge.net/mingw-w64/x86_64-$packageVersion-release-$threads-$exceptionHandling64-rt_v3-$rev.7z"
+#http://downloads.sourceforge.net/mingw-w64/x86_64-4.8.3-release-posix-sjlj-rev0.7z
```
Comment (ferventcoder) [inline manual/mingw/tools/chocolateyInstall.ps1]: Whats the diff?

Comment (ferventcoder) [inline manual/mingw/tools/chocolateyInstall.ps1]: I mean besides one is shorter and has no spaces.

Comment (MikaelSmith) [inline manual/mingw/tools/chocolateyInstall.ps1]: That's it. I can not bother, but I think projects can re-arrange their
directory structure but the short link will still work.

Comment (ferventcoder) [inline manual/mingw/tools/chocolateyInstall.ps1]: Sweet. Will fix up and republish.

Comment (ferventcoder) [inline manual/mingw/tools/chocolateyInstall.ps1]: Not fix up this, fix up the package

**Pull request merged**